frontend/public/style.css

```diff
@@ -13,6 +13,7 @@ body {
   display: flex;
   justify-content: center;
   align-items: center;
+  user-select: none;
```

Comment:
I assume this change is related to Issue #41? As changes start getting larger you'll want to separate out tickets into their own branches and their own pull requests. For two small changes like this it's trivial, but if you start combining large features into a single pull request it can get confusing fast, not to mention hard to understand if you look back at the project's history later. It's cool now, just keep that in mind for the future!

Reply:
Ope, I see you already removed it before I could finish my review! Very nice!